src/main/java/bio/terra/cda/app/util/QueryTranslator.java

```diff
@@ -19,6 +20,15 @@ public static String sql(String table, Query query) {
     return new SqlGenerator(table, query).generate();
   }
 
+  /** Split the TABLE.ColumnName into component parts */
+  public static Map<String, String> parseTableName(String qualifiedTable) {
```

Comment:
Having a public function that returns `Map` is not great since a caller will need to know what names the map contains and what type each element is. Either return a class here or split it into two methods. Or inline the code directly in the caller, as is done elsewhere in this class.

Reply:
I created a new model NestedColumn that will return an object.